cinema_service/settings.py

```diff
@@ -42,6 +42,7 @@
     "rest_framework",
     "cinema",
     "user",
+    "debug_toolbar",
```

Comment:
Consider making the inclusion of `debug_toolbar` in `INSTALLED_APPS` conditional based on the `DEBUG` setting to avoid unnecessary overhead in production.